logstash-output-kafka.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-kafka'
-  s.version         = '6.1.4'
+  s.version         = '6.1.5'
```

Comment:
since this is a new feature and not just a bug fix. I'd be happy to bump this to `6.2.0`